workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/WorkflowInterceptorTest.kt

```diff
@@ -198,7 +200,7 @@ internal class WorkflowInterceptorTest {
         key: String,
         sideEffect: suspend CoroutineScope.() -> Unit
       ) {
-        runBlocking { sideEffect() }
+        launch { sideEffect() }
```

Comment:
where do we ensure that this gets run? Here and elsewhere you could be using `runTest(UnconfinedTestDispatcher()` if we want everythign to go ahead eagerly.

Reply:
Ah I might have missed this one, but I've added `advanceUntilIdle()` where these are required, I should add this here as well to prevent confusion.

But I've confirmed this isn't required in this particular test as though this could be ran in parallel, the render()+assertion still always happens at the end of the test in the single thread.

Does that make sense? Though it's possible I might be reading this wrong https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-test.html